styleguide/source/_patterns/02-organisms/00-global/search-results-columns.twig

```diff
@@ -0,0 +1,11 @@
+<div class="layout layout_two-wrap search-results-columns">
+ 
+     {% include 'molecules-search-result-external' with {'class': 'layout_two-wrap-item'} %}
```

Comment:
Should the classes on the items here be search-results-columns-item in addition to layout_two-wrap-item in order to follow the class pattern that search-results-list-item uses?